openfisca_france/parameters/prestations/prestations_familiales/af/modulation/plafond_tranche_1.yaml

```diff
@@ -23,6 +23,8 @@ values:
   2019-01-01:
     value: 68217.0
     reference: https://www.legifrance.gouv.fr/eli/arrete/2018/12/21/SSAS1835267A/jo/article_1
-  2020-04-01:
+  2020-01-01:
     value: 69309.0
     reference: https://www.legifrance.gouv.fr/eli/arrete/2019/12/20/SSAS1934813A/jo/article_1
+  2021-01-01:
+    value: 69933.0
```

Comment:
Il manque un retour à la ligne ([configurer vscode pour qu'il le fasse à ta place à chaque sauvegarde](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)).

Cela peut également valoir le coup d'activer [l'option qui enlève les espaces en bout de ligne ](https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut).

@NolwennLoisel

Reply:
J'ai corrigé le fichier en ajoutant la ligné. @benjello